src/librustdoc/lib.rs

```diff
@@ -536,6 +536,7 @@ fn opts() -> Vec<RustcOptGroup> {
                 "[unversioned-shared-resources,toolchain-shared-resources,invocation-specific]",
             )
         }),
+        unstable("no-run", |o| o.optflag("", "no-run", "Compile doc tests without running it")),
```

Comment:
```suggestion
        unstable("no-run", |o| o.optflag("", "no-run", "Compile doctests without running them")),
```
Other arguments use the single-word form and treat it as a plural, better to keep consistency here.